app/views/main_controller/index.html.slim

```diff
@@ -0,0 +1,30 @@
+.top-section.flex.full-width
+  = image_tag("phone.png", :class => ["phone", "margin-20"])
+  .space-bar.margin-20
+  = image_tag("tcg_logo.png", :class => ["logo", "margin-20"])
+.middle-section.flex.flex-wrap.full-width
+  .icon-container.flex-column
+    = image_tag("ic_list.png", :class => "icon")
+    span.icon-title
```

Comment:
Why are these spans? You don't want them to be inline elements, you are going to manage them with flex